test/test_utils.py

```diff
@@ -589,3 +589,35 @@ def test_does_not_output_when_passed_true(self, mock_click):
 
         # ASSERT
         mock_click.echo.assert_not_called()
+
+
+class TestIsValidDefinitionFile(TestCase):
+    def test_true_returned_if_correct_definition_file_type(self):
+        # SETUP
+        file_name = Path("path/to/definition.yml")  # only tests for .yml
```

Comment:
This could be modified to use a for loop to test each of the valid options in a single test.